.gitignore

```diff
@@ -20,3 +20,6 @@ out/
 sbt-launch-0.7.5.jar
 sbt-launch.jar
 *.pyc
+.project
```

Comment:
please don't add these (I think these are eclipse files?) can you instead add these to your global gitignore?